apperals-frontend/src/components/navigation/ButtonRow.js

```diff
@@ -18,12 +24,43 @@ function ButtonRow() {
             <Link to="/page2">
               <button>Page 2</button>
             </Link>
-            <button>Button 3</button>
-            <button>Button 4</button>
-            <button>Button 5</button>
+            <Link to="/page3">
+              <button>Page 3</button>
+            </Link>
+            <Link to="/page4">
+              <button>Page 4</button>
+            </Link>
+            <Link to="/page5">
+              <button>Page 5</button>
+            </Link>
+
+            {/* only shows when screen shrinks */}
+            <div className="hamburger-icon" onClick={toggleMenu}>
+              &#9776; {/* Unicode for hamburger icon */}
+            </div>
           </div>
         </div>
 
+        {/* Mobile menu for hamburger */ }
+        <div className={`mobile-menu ${showMenu ? 'open' : ''}`}>
+          <Link to="/page1">
+            <button>Page 1</button>
+          </Link>
+          <Link to="/page2">
+            <button>Page 2</button>
+          </Link>
+          <Link to="/page3">
+            <button>Page 3</button>
+          </Link>
+          <Link to="/page4">
+            <button>Page 4</button>
+          </Link>
+          <Link to="/page5">
+            <button>Page 5</button>
+          </Link>
+        </div>
+
+        {/* container for background image under nav bar */ }
         <div class="image-container">
```

Comment:
We can create another Component for this functionality / Banner Image. This would be a part of the Landing Page. Move this to another Component (eg : Banner)